devfiles/gradle-image/devfile.yaml

```diff
@@ -46,7 +46,7 @@ commands:
                       echo "The component was already built. The devBuild command is not run." && exit 0;
                     else
                       echo "Running devBuild command ..." &&
-                      gradle -Dgradle.user.home=/home/user/.gradle assemble -Pliberty.runtime.version=21.0.0.9 &&
+                      gradle -Dgradle.user.home=/home/user/.gradle assemble -Pliberty.runtime.version=21.0.0.12 &&
```

Comment:
At some point we should probably switch to a variable for version .  Doesn't have to be now.